actors/builtin/miner/miner_actor.go

```diff
@@ -1875,8 +1875,8 @@ func getVerifyInfo(rt Runtime, params *SealVerifyStuff) *abi.SealVerifyInfo {
 	err = rt.Message().Receiver().MarshalCBOR(buf)
 	AssertNoError(err)
 
-	svInfoRandomness := rt.GetRandomness(crypto.DomainSeparationTag_SealRandomness, params.SealRandEpoch, buf.Bytes())
-	svInfoInteractiveRandomness := rt.GetRandomness(crypto.DomainSeparationTag_InteractiveSealChallengeSeed, params.InteractiveEpoch, buf.Bytes())
+	svInfoRandomness := rt.GetRandomnessFromTickets(crypto.DomainSeparationTag_SealRandomness, params.SealRandEpoch, buf.Bytes())
+	svInfoInteractiveRandomness := rt.GetRandomnessFromTickets(crypto.DomainSeparationTag_InteractiveSealChallengeSeed, params.InteractiveEpoch, buf.Bytes())
```

Comment:
This would be `GetRandomnessFromBeacon`

Reply:
I thought we said all the PoRep stuff was using the vrf chain?

Reply:
No, there is an analysis that shows that we could, but we should rather not do it. It reduces the PoRep security and it changes the way the VRF chain is generated.

Reply:
I "be the change you want to see"d